README.md

```diff
@@ -24,3 +24,24 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
```

Comment:
Couldn't figure out how to leave comments on binary files, so will leave them here. Add READMEs in subdirs that documents what the different files are. Also consider renaming some of them. shotgun_dialogue is some naming inherited from a completely different dataset.

Reply:
I think the shogun_dialogue files should be either removed from the public repo or first converted into another format. These files are Protos that are not publicly accessible and therefore would confuse people who look at them.

Reply:
I made this into an issue. I'll have to reorganize how I'm building the movielens data, and I think that should be on the next PR because that has the rest of the movielens stuff.